webpack.config.js

```diff
@@ -74,11 +74,6 @@ module.exports = {
 
 if (nodeEnv === 'watch') {
   module.exports.watch = true;
-  module.exports.watchOptions = {
```

Comment:
Will it affect watch performance if removing those `ignored` configuration?

Reply:
I added this before to improve performance but i don't think it makes diff. We already have include in tsconfig so it is bit redundancy. We still need to find out how we can improve dev build performance.